packages/cx/src/charts/Legend.js

```diff
@@ -86,11 +92,11 @@ export class Legend extends HtmlElement {
       return [list, super.renderChildren(context, instance)];
    }
 
-   renderShape(entry) {
+   renderShape(entry, instance) {
```

Comment:
Don't pass the whole `instance` object here. Just add a parameter called `shape` and pass it from the top.